### DISCUSSION TITLE
Azteca Guatemala

### DISCUSSION THREAD

**Discussion by R4IN2007** [iptv-org/community#454] [Show and tell]
### Channel ID

AztecaGuatemala.gt

### Stream URL

https://tvaztecaguate.com/en-vivo/ 

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by Sphinxroot** [comment]
https://5fe2654d6127d.streamlock.net/aztecaguate/videoaztecaguate/playlist.m3u8?wmsAuthSign= base64 token
decoded
server_time=9/29/2022 4:26:40 PM&hash_value=xxxxxxxxxxxxxxxxxxx==&validminutes=5&id=

<html><body>


referrer | https://conceptoweb-studio.com/radio/video/aztecaguate/
-- | --



</body>
</html>

**Comment by BellezaEmporium** [comment]
http://free.fullspeed.tv/iptv-query?streaming-ip=https://conceptoweb-studio.com/radio/video/aztecaguate/

**Comment by BellezaEmporium** [comment]
https://q-s-l4n3sh4d0w.onrender.com/iptv-query?streaming-ip=https://conceptoweb-studio.com/radio/video/aztecaguate/

**Comment by AntiPontifex** [comment]
Thank you for the stream: it will be included in the next update.

**Comment by freearhey** [comment]
http://free.fullspeed.tv/iptv-query?streaming-ip=https://conceptoweb-studio.com/radio/video/aztecaguate/ (offline)
<img width="745" alt="image" src="https://user-images.githubusercontent.com/7253922/232235480-8c2357ae-5a1b-4e72-bdba-add781968718.png">

https://q-s-l4n3sh4d0w.onrender.com/iptv-query?streaming-ip=https://conceptoweb-studio.com/radio/video/aztecaguate/ (offline)
<img width="830" alt="image" src="https://user-images.githubusercontent.com/7253922/232235501-87764454-b32c-4735-91b9-41eee18355e6.png">

**Comment by R4IN2007** [comment]
Stream works fine in Chrome
![Screenshot_20230609_193434_Chrome](https://github.com/iptv-org/iptv/assets/83226646/bd261aa1-f7e4-4b03-8e6b-440999ca46a6)

**Comment by BellezaEmporium** [comment]
> Stream works fine in Chrome
> ![Screenshot_20230609_193434_Chrome](https://github.com/iptv-org/iptv/assets/83226646/bd261aa1-f7e4-4b03-8e6b-440999ca46a6)
> 
> 

My implementation's offline since some time ago. One can simply fork the query-streamlink repo to get the result needed.
